src/main/java/com/faforever/client/filter/LiveGamesFilterController.java

```diff
@@ -90,4 +109,16 @@ public GameType fromString(String string) {
       throw new UnsupportedOperationException("Not supported");
     }
   };
+
+  @Override
+  protected void afterBuilt() {
+    hideModdedGamesFilter.valueProperty().bindBidirectional(preferences.getVault().getLiveReplaySearch().getHideModdedGames());
+    hideSingleGamesFilter.valueProperty().bindBidirectional(preferences.getVault().getLiveReplaySearch().getHideSingleGames());
+    onlyGamesWithFriendsFilter.valueProperty().bindBidirectional(preferences.getVault().getLiveReplaySearch().getOnlyGamesWithFriends());
+    onlyGeneratedMapsFilter.valueProperty().bindBidirectional(preferences.getVault().getLiveReplaySearch().getOnlyGeneratedMaps());
+    gameTypeFilter.valueProperty().bindBidirectional(preferences.getVault().getLiveReplaySearch().getGameTypes());
+    featuredModFilter.valueProperty().bindBidirectional(preferences.getVault().getLiveReplaySearch().getModName());
+    playerFilter.valueProperty().bindBidirectional(preferences.getVault().getLiveReplaySearch().getPlayerName());
```

Comment:
To get around the nesting and chained method calls in PreferencesConfig the nested preferences are exposed as their own beans so they can be injected directly

Reply:
As requested, it is now injected as bean even though it is still nested.